### PR TITLE
Moves single-use extensions from isolated file to caller file

### PR DESCRIPTION
**Pull request by esreli**
_Note, no code change is performed in this PR._ This change is progressively performed to clean up the project structure by moving existing single-use extensions from their isolated file to the caller file.

### CODE REVIEW

**Comment by philium** [comment]
For those that you can (like `AGSMap`), I would suggest making the extension `private`. As for the other extensions (like those that add conformance to a protocol), they can't be made private, which is a good argument for keeping them in separate source files. Since they are non-private, they could end up being used elsewhere at any time, making them non-single-use. Just a thought.

**Comment by esreli** [comment]
>  I would suggest making the extension `private`.

This is a good call.

**Comment by esreli** [comment]
@mhdostal @philium i'm marking this first batch as ready for review. Would you have a look and offer your feedback? Thank you in advance.

File: data-collection/data-collection/App Context/OfflineMapManager.swift [hunk 1]
```diff
@@ -390,3 +390,17 @@ fileprivate extension URL {
 protocol JobResult {}
 extension AGSGenerateOfflineMapResult: JobResult {}
 extension AGSOfflineMapSyncResult: JobResult {}
+
+
+extension AGSMap {
```
Comment (philium) [inline data-collection/data-collection/App Context/OfflineMapManager.swift:395]:
```suggestion
private extension AGSMap {
```

File: data-collection/data-collection/Extensions/ArcGIS/AGSArcGISFeatureTable/AGSArcGISFeatureTable+Query.swift [hunk 2]
```diff
@@ -162,3 +162,16 @@ extension AGSArcGISFeatureTable {
         }
     }
 }
+
+// MARK: Query Parameters
+
+private extension AGSQueryParameters {
+    
+    /// Return a new `AGSQueryParameters` with a `whereClause` requesting all features in the table.
+
```
Comment (philium) [inline data-collection/data-collection/Extensions/ArcGIS/AGSArcGISFeatureTable/AGSArcGISFeatureTable+Query.swift:171]:
```suggestion
```

**Review by philium — APPROVED** [review]

**Review by philium — APPROVED** [review]

**Comment by esreli** [comment]
@mhdostal have you had a chance to review this PR? what do you think?

**Pull request merged**